test/Helpers/LeanCode.UserIdExtractors.Tests/UserIdExtractorsTests.cs

```diff
@@ -0,0 +1,105 @@
+using System.Reflection;
+using System.Security.Claims;
+using LeanCode.DomainModels.Ids;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace LeanCode.UserIdExtractors.Tests;
+
+[TypedId(TypedIdFormat.RawInt)]
+public readonly partial record struct TestIntId;
+
+[TypedId(TypedIdFormat.RawLong)]
+public readonly partial record struct TestLongId;
+
+[TypedId(TypedIdFormat.RawGuid)]
+public readonly partial record struct TestGuidId;
+
+[TypedId(TypedIdFormat.PrefixedUlid, CustomPrefix = "tpl")]
+public readonly partial record struct TestPrefixedUlidId;
+
+[TypedId(TypedIdFormat.PrefixedGuid, CustomPrefix = "tpg")]
+public readonly partial record struct TestPrefixedGuidId;
+
+public class UserIdExtractorsTests
+{
+    private const string UserIdClaim = "sub";
+
+    private readonly IServiceProvider serviceProvider;
+
+    public UserIdExtractorsTests()
+    {
+        var services = new ServiceCollection();
+
+        services.AddUserIdExtractors([ Assembly.GetExecutingAssembly() ], UserIdClaim);
```

Comment:
Nitpicking (something for another PR most likely) but there should be no spaces between `[` and the first element (and between the last element and `]`), this is a CSharpier bug that has been fixed in some recent release. So we will need a CSharpier bump, probably sooner than later.

Reply:
Will bump it, probably not in this PR though